low-level/files/opt/seagate/sspl/setup/setup_sspl

```diff
@@ -72,7 +72,7 @@ then
 fi
 
 # configure sspl
-$SSPL_BASE_DIR/bin/sspl_post_install -p "$PRODUCT_NAME"
```

Comment:
why this setup_sspl shell script  needed still? Going to create confusions with sspl_setup.py

Reply:
Removed. Earlier we used this with sspl_dev_deploy.sh which runs all sspl setup commands at once.
Now this is being replaced by sspl_setup.py

Reply:
Also, service related validations are covered under sspl_setup Check.